extension/src/commands/util.ts

```diff
@@ -5,7 +5,14 @@ import { Context } from '../vscode/context'
 
 export const showSetupOrExecuteCommand =
   <T>(setup: Setup, callback: (context: Context) => Promise<T | undefined>) =>
-  (context: Context) =>
-    setup.shouldBeShown()
-      ? commands.executeCommand(RegisteredCommands.SETUP_SHOW_DVC)
-      : callback(context)
+  (context: Context) => {
+    if (!setup.isDvcSetup()) {
+      return commands.executeCommand(RegisteredCommands.SETUP_SHOW_DVC)
+    }
+
+    if (!setup.isExperimentsSetup()) {
+      return commands.executeCommand(RegisteredCommands.SETUP_SHOW_EXPERIMENTS)
+    }
+
```

Comment:
Another option is to just open `SETUP_SHOW`, keeping all sections open no matter what is setup. I chose to write checking logic instead since I don't think it's overly complex and gets the user to where they need to go to get started with the extension a little faster :)